xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-core/src/main/java/org/xwiki/annotation/Annotation.java

```diff
@@ -84,6 +84,16 @@
      */
     public static final String TARGET_FIELD = "target";
 
+    /**
+     * The name of the field of this annotation start offset relative to the plain text.
+     */
```

Comment:
Don't forget to indicate `@since` when this new API is available before merging it.